Source/ColouringBook/ColouringBook.h

```diff
@@ -8,7 +8,7 @@
 //
 // Place any includes here that are needed by the majority of your .cpp files
 
-#include "EngineMinimal.h"
+#include "Engine.h"
```

Comment:
I guess that the main drawback could be compiling time? 

Back to including EngineMinimal.h instead Engine.h